src/algoliasearch.helper.js

```diff
@@ -1331,7 +1331,7 @@ AlgoliaSearchHelper.prototype.clearCache = function() {
 AlgoliaSearchHelper.prototype.setClient = function(newClient) {
   if (this.client === newClient) return this;
 
-  if (newClient.addAlgoliaAgent && !doesClientAgentContainsHelper(newClient)) newClient.addAlgoliaAgent('JS Helper ' + version);
+  if (newClient.addAlgoliaAgent && !doesClientAgentContainsHelper(newClient)) newClient.addAlgoliaAgent('JS Helper (' + version + ')');
```

Comment:
Can we avoid inlining `if` statements?

Reply:
I can change this but the entire lib is using this. Does it make sense to just change it in one place?

Reply:
As edited in [the first UA change](https://github.com/algolia/algoliasearch-helper-js/pull/647/files#diff-cbf795d639d5f4b610b6d0f6ee395b2aR123), we don't use an inline condition. We should respect that rule for edited code IMO.

Reply:
Done